src/components/ConditionalFilter/constants.ts

```diff
@@ -76,7 +76,7 @@ export const STATIC_OPTIONS: LeftOperand[] = [
   },
   {
     value: "giftCard",
-    label: "Has giftcard",
+    label: "Is giftcard",
```

Comment:
**question**: shouldn't labels be translated?

Reply:
it will, however we can't translate all of them since they are dynamic